docs/Quantiles.ipynb

```diff
@@ -0,0 +1,562 @@
+            "text/plain": [
+              "<Figure size 900x600 with 1 Axes>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "#make the transformation get discrete scores per candidate\n",
+        "discrete_scores = dp.t.make_quantile_score_candidates(dp.vector_domain(dp.atom_domain(T=int)), \n",
+        "                                    dp.symmetric_distance(), \n",
+        "                                    candidates, \n",
+        "                                    quantile)\n",
+        "#get the ages from the dataset and change its type for the transformation\n",
+        "age_list = list(df.select(\"AGE\").collect().get_column(\"AGE\").to_numpy().astype('int32'))\n",
+        "#apply the transformation to the discrete scores\n",
+        "discrete_scores_list = discrete_scores(age_list)\n",
+        "#plot the scores\n",
+        "fig, ax = plt.subplots(figsize = (9, 6))\n",
+        "ax.hist(discrete_scores_list, bins=15, edgecolor=\"black\");"
```

Comment:
I'd like to always have axis labels.

Reply:
Can you discuss what we can expect to see in the graph and what it means? I was expecting this to be a "v" shape, with the minimum in the middle, but that's not the case.

There's also a `1e9` in the lower right of the figure. Is that a multiplier for the horizontal axis?